docs/config.md

```diff
@@ -131,12 +131,14 @@ FIELDS = [
     {'key': 'name', 'boost': 4, 'null': False},
     {'key': 'street'},
     {'key': 'postcode',
-     'boost': lambda doc: 1.2 if doc.get('type') == 'commune' else 1},
+     'boost': lambda doc: 1.2 if doc.get('type') == 'municipality' else 1},
```

Comment:
@yohanboniface @cquest can you please confirm that `commune` isn't the good type anymore, but that's `municipality`? Can you please provide the types list? It's probably worth adding it to the documentation.

Reply:
The `type` does not belongs to addok, but to the data source you use :)
In you case, it's the BAN, which comes from https://github.com/etalab/ban-data/ :)

Reply:
Alright so `municipality` is indeed the good type. As the documentation is written for the BAN, I think this change is okay. I'm going to update this PR to indicate that the `type` is from the data source.